runelite-client/src/main/java/net/runelite/client/plugins/screenmarkers/ScreenMarkerOverlay.java

```diff
@@ -47,6 +47,11 @@
 		setLayer(OverlayLayer.ALWAYS_ON_TOP);
 		setPriority(OverlayPriority.HIGH);
 
+		update();
+	}
+
+	public void update()
+	{
 		screenMarkerRenderable = new ScreenMarkerRenderable();
```

Comment:
Instead of this I would just set all the properties in the render method, and leave the renderable final and initialize when declaring